x-pack/plugin/esql/qa/testFixtures/src/main/resources/match-function.csv-spec

```diff
@@ -184,6 +184,19 @@ emp_no:integer | first_name:keyword | last_name:keyword
 10043          | Yishay             | Tzvieli      
 ;
 
+testMultiValuedFieldWithConjunctionWithScore
+required_capability: match_function
+required_capability: metadata_score
+
+from employees METADATA _score
```

Comment:
Nitty nit: "metadata" is a keyword, like the others ("from", "where" etc.). I guess we want to EMPHASISE it, but in case we'll add any of these queries to the docs as examples, we might want to make it case-consistent.

Reply:
related to the existing lack of capitalization in this test file - https://github.com/elastic/elasticsearch/pull/116399